articles/openshift/openshift-faq.md

```diff
@@ -10,196 +10,220 @@ ms.date: 11/04/2019
+Nodes are rebooted as a part of an upgrade.
 
-+ Backup and management of etcd data is an automated process that may require cluster downtime depending on the action. If the etcd database is being restored from a backup there will be downtime. We back up etcd hourly and retain the last 6 hours of backups.
+## Cluster operations
 
-+ Cloud provider-initiated maintenance includes network, storage, and regional outages. The maintenance is dependent on the cloud provider and relies on provider-supplied updates.
+### Can I use Prometheus to monitor my applications?
 
-## What is the general upgrade process?
+Yes, you can deploy Prometheus in your namespace and monitor applications in your namespace.
 
-Running an upgrade should be a safe process to run and should not disrupt cluster services. The SRE can trigger the upgrade process when new versions are available or CVEs are outstanding.
+### Can I use Prometheus to monitor metrics related to cluster health and capacity?
 
-Available updates are tested in a stage environment and then applied to production clusters. When applied, a new node is temporarily added and nodes are updated in a rotating fashion so that pods maintain replica counts. Following best practices helps ensure minimal to no downtime.
+Yes in Azure Red Hat OpenShift 4.x.
 
-Depending on the severity of the pending upgrade or update, the process might differ in that the updates might be applied quickly to mitigate the service’s exposure to a CVE. A new image will be built asynchronously, tested, and rolled out as a cluster upgrade. Other than that, there is no difference between emergency and planned maintenance. Planned maintenance is not prescheduled with the customer.
+No in Azure Red Hat OpenShift 3.11.
 
-Notifications may be sent via ICM and email if communication to the customer is required.
+### Can logs of underlying VMs be streamed out to a customer log analysis system?
 
-## What about emergency vs. planned maintenance windows?
+Logs from underlying VMs are handled by the managed service and are not exposed to customers.
 
-We do not distinguish between the two types of maintenance. Our teams are available 24/7/365 and do not use traditional scheduled “out-of-hours” maintenance windows.
+### How can a customer get access to metrics like CPU/memory at the node level to take action to scale, debug issues, etc. I cannot seem to run kubectl top on an Azure Red Hat OpenShift cluster.
 
-## How will host operating system and OpenShift software be updated?
+For Azure Red Hat OpenShift 4.x clusters, the web console contains all metrics at the node level. See the Red Hat documentation on [viewing cluster information](https://docs.openshift.com/aro/4/web_console/using-dashboard-to-get-cluster-information.html) for more details.
```

Comment:
specify which web console. The OpenShift web console, right?

Reply:
Good point. I clarified this.